tchannel/response.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from . import transport
+
+__all__ = ['Response']
+
+
+class Response(object):
+    def __init__(self, header, body, transport):
+        self.header = header
```

Comment:
@blampe this is actually a weird thing because for `raw` it's a free-form "header block", but for `json` & `thrift` it is basically key-value pairs... I suppose `headers` is more right for the majority, even if `raw` it would be free-form. 

@abhinav @junchaowu what do you guys think?